Exiled.DedicatedNetwork/GlobalSuppressions.cs

```diff
@@ -0,0 +1,12 @@
+// -----------------------------------------------------------------------
```

Comment:
Why are any of these suppressions necessary? There is a reason these rules exist, and they should be followed in all Exiled.x projects.